scripts/build.sh

```diff
@@ -17,6 +17,7 @@
 set -ev
 
 npm run lint
+npm run docs
```

Comment:
Added this so travis will build docs on normal PRs so it will catch docs issues before they make their way into the repo.